src/dotenv.cr

```diff
@@ -1,107 +1,17 @@
-module Dotenv
-  extend self
-
-  class ParseError < Exception
-  end
-
-  # Raises an exception on parsing error.
-  class_property strict : Bool = true
+require "./parser"
 
+module Dotenv
   @[Flags]
```

Comment:
Why do we have `@[Flags]`? I read the reference but it's not clear to me what the benefits are https://crystal-lang.org/reference/syntax_and_semantics/enum.html

Reply:
It's to have the `None` value